news/2020-01-05-beatmap-spotlights-autumn-2019.md

```diff
@@ -0,0 +1,365 @@
+
+[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/osu/sit.jpg)](https://osu.ppy.sh/beatmapsets/819752)
+
+[Mirash](https://osu.ppy.sh/users/2841009) has seen a fair amount of airtime in the osu! spotlights, and Sit down. Stand up. (Snakes & Ladders.) earns him his second nominated mapset this season. Though some may criticize the map as "boring" for the most part, Mirash manages to do an excellent job of creating an atmosphere unlike much of what's been seen before. The map is a gradual crescendo culminating in a fiery, chaotic payoff that mirrors the song's central themes of oppression, control, and uprising. 
+
+The beauty in Sit down. Stand up. (Snakes & Ladders.) stems from the story it tells, from beginning to end.
+
+_written by [Toy](https://osu.ppy.sh/users/2757689)_
+
+[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/osu/instrumental.jpg)](https://osu.ppy.sh/beatmapsets/935839)
+
+Being a fanatic of Nekomata Master’s music tracks, [Icekalt](https://osu.ppy.sh/users/5410645) decided to compile some of their best in Instrumental Compilation. He portrays every track excerpt as thoroughly as possible, putting in 1/6 trills, long chains of low bpm streams, varying slider velocity choices constantly, all in an effort to keep the map as engaging as possible.
+
+On top of this, he also invited mappers [ScubDomino](https://osu.ppy.sh/users/8972308), [den0saur](https://osu.ppy.sh/users/5385151), [Xilver15](https://osu.ppy.sh/users/3099689) and [celerih](https://osu.ppy.sh/users/4696296) to map a few of the songs as well, with each of their own unique styles to represent their respective songs.
+
+If you have enough time, be sure to check this one out!
+
+_written by [KnightC0re](https://osu.ppy.sh/users/7894340)_
+
+[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/osu/last.jpg)](https://osu.ppy.sh/beatmapsets/1015203)
+
+Stop arguing over whether flawless playability and high-end aesthetic can exist in the same map - [captin1](https://osu.ppy.sh/users/689997) and [Kalibe](https://osu.ppy.sh/users/3376777) have proven in their collaboration work, lmy - Last words, that it is completely possible. Multiple takes on emphasis combining with positively weird but well-designed slider patterns bring a whole new playing experience set to Aitsuki Nakuru's beautiful voice.
+
+_written by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)_
+
+[![](/wiki/shared/news/2020-01-05-beatmap-spotlights-autumn-2019/osu/natsu.jpg)](https://osu.ppy.sh/beatmapsets/1006502)
+
+Coupled with some amazing keysounding, the top difficulty uses numerous equal-distance obtuse note patterns to create a flow with slightly increasing difficulty to accurately represent the little crescendos in the piano melodies, and makes ample use of various distance gimmicks under low approach rate plus clearly distinguishable time-distance gaps to maximize the representation of the song's atmosphere in all of its different parts. 
```

Comment:
This run on sentence gets really difficult to read. There's way too much going on, the different points should be seperated into sentences where fitting - this would require rewriting.

Reply:
No idea how to separate this as I have zero clue on the map itself

Reply:
Hopefully fixed